network-api/networkapi/templates/pages/buyersguide/catalog.html

```diff
@@ -11,18 +11,10 @@
   <input type="hidden" class="parent-title" value="{{current_category.parent.localized.name}}">
   <div class="tw-container">
     <div class="tw-row">
-      <div class="tw-px-4 tw-w-full">
-        <div id="sticky-bar" class="creepiness-slider bg-white text-center tw-flex tw-justify-center">
-          <div class="creep-o-meter-information">
-            <p class="speech-bubble-container">
-              <span class="speech-bubble tw-bg-gradient-to-t tw-from-purple-05 tw-to-blue-05">
-                <img class="hook" src="{% static "_images/buyers-guide/icon-bubble-hook.svg" %}">
-                <span class="text">{% trans "We created this guide to help you shop<br> for safe, secure connected products." %}</span>
-              </span>
-            </p>
-            <span class="current-creepiness"></span>
-          </div>
-        </div>
+      <div class="tw-px-4 tw-py-5 large:tw-py-7 tw-w-full">
+        <span class="tw-block tw-font-zilla tw-font-light tw-text-center tw-text-2xl large:tw-text-4xl tw-m-0">
+          {% trans "Make smart choices to protect your privacy. <br /> Search for products. Read expert reviews. Get tips and tricks." %}
+        </span>
```

Comment:
Can you add some context why we use `span` with `tw-block` instead of `div`? I have seen this elsewhere too. I am just wondering if I am missing something about that.

Reply:
Spans are not display block by default unlike other elements, so text-align does not work for inline elements! 

Good to know about the force pushes!

Reply:
I do not think there is much different from div/spans but I like to use spans for text-content so its easier to use querySelector stuff in the future so I don't have to add unique ids/classes. I honestly think both are fine and it is just preference but I should look up which is more semantic.

Reply:
> Spans are not display block by default unlike other elements, [...]

That's why I was wondering why not just use a `div` if we want it to be a block element anyways 🤔

Reply:
> I do not think there is much different from div/spans but I like to use spans for text-content so its easier to use querySelector stuff in the future so I don't have to add unique ids/classes. I honestly think both are fine and it is just preference but I should look up which is more semantic.

Thanks for expanding on that 👍